README.md

```diff
@@ -1,57 +1,20 @@
-[![Build Status](https://secure.travis-ci.org/eBay/UAF.svg?branch=master)](https://travis-ci.org/eBay/UAF)  [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/eBay/UAF)
-# UAF - Universal Authentication Framework
+[![Build Status](https://secure.travis-ci.org/neXenio/UAF.svg?branch=master)](https://travis-ci.org/eBay/UAF)
 
-[UAF Architectural Overview](https://fidoalliance.org/wp-content/uploads/html/fido-uaf-overview-v1.0-ps-20141208.html)
+# FIDO UAF - Universal Authentication Framework
 
-[News & Release Notes](RELEASE.md)
+This projects provides a Java implementation of the [UAF protocol][uaf-protocol-overview] (as defined per [specification][uaf-specification-v1.0]) that can be used by [UAF Clients][uaf-client-overview] and [UAF Servers][uaf-server-overview].
 
-- [2017.10.02 - Updated Android RP app to utilize Android 6.0 secure storage](https://github.com/eBay/UAF/commit/cabc4be435b9c24ed7ac4f677b5e1bfce73a379d)
-- [2016.10.14 - Added endpoints for whitelisting AAID, and Facet Ids](https://github.com/eBay/UAF/blob/master/RELEASE.md#20161014)
-- [2016.05.20 - Added wiki page for Maven build and Tomcat setup/run (using CLI only)](https://github.com/eBay/UAF/wiki/BuildingAndRunningUAFServerUsingMaven(CLIonly))
-- [2016.03.28 - Started wiki pages](https://github.com/eBay/UAF/wiki)
-- [2016.02.23](https://github.com/eBay/UAF/blob/master/RELEASE.md#20160223)
-- [2016.01.16](https://github.com/eBay/UAF/blob/master/RELEASE.md#20160116)
 
-## Vision
-The main goal is the passwordless authentication experience
+## What is UAF?
 
-## Values
-- Simple to authenticate using biometrics readings, such as fingerprint
-- More secure authentication using the cryptography
+The goal of the Universal Authentication Framework is to provide a unified and extensible authentication mechanism that supplants passwords while avoiding the shortcomings of current alternative authentication approaches. Read more at the [UAF Specification][uaf-specification-v1.0].
 
```

Comment:
I think we should add a reference to ebays original implementation, give attribution, and explain the fork.